tools/tools.go

```diff
@@ -25,8 +25,8 @@ import (
 //go:generate go install github.com/evanphx/grpc-gateway/protoc-gen-swagger
 	_ "github.com/evanphx/grpc-gateway/protoc-gen-swagger"
 
-//go:generate go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
-	_ "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway"
+//go:generate go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
+	_ "github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway"
```

Comment:
This is the only thing that actually changed - everything else is the result of `go mod tidy` and `make gen/server`